netbout-client/src/main/java/com/netbout/mock/MkBout.java

```diff
@@ -101,13 +102,17 @@ public Date date() throws IOException {
 
     /**
      * {@inheritDoc}
-     * @todo #806:30min/DEV This method needs to be implemented
```

Comment:
@krzyk ticket #805 has been created first  and then the puzzle #827 
I missed it
If i removed this todo I guess #827 will be closed 
no one assigned to that puzzle